Loungeware/objects/username_gamename_obj_button/Step_0.gml

```diff
@@ -0,0 +1,15 @@
+if (KEY_ANY_PRESSED) {
+	//see jsdoc for microgame_win()
```

Comment:
I think that having a comment that says to read something else so minimally is not a good comment. I know it seems redundant but perhaps we can expand this. 

"Whenever the player performs the action that should mark the game has having been 'won', you should call this function. It's okay to call it more than once, and it's also okay to toggle back and forth. "